main.js

```diff
@@ -3,71 +3,65 @@ var cookpotImage = document.querySelector('.cookpot');
 var dishPrinted = document.querySelector('.dish-output');
 var dishHeading = document.querySelector('.dish-heading');
 var clearButton = document.querySelector('.clear');
+var radioInputs = document.getElementsByName('food');
 
-// var currentSide = sides[getRandomIndex(sides)];
-// var currentMain = mains[getRandomIndex(mains)];
-// var currentDessert = desserts[getRandomIndex(desserts)];
+letsCookButton.addEventListener('click', function() {
+  showRandomDish();
+  showDishPage();
+});
 
-letsCookButton.addEventListener('click', showRandomDish);
-clearButton.addEventListener('click', showHomePage);
+clearButton.addEventListener('click', function() {
+  showHomePage();
+  clearRadioButtons();
+});
 
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 };
 
+function showDishPage() {
+  event.preventDefault();
+  cookpotImage.classList.add('hidden');
+  dishPrinted.classList.remove('hidden');
+  dishHeading.classList.remove('hidden');
+  clearButton.classList.remove('hidden');
+};
+
 function showRandomDish() {
+  dishPrinted.innerText = '';
   var selectedButton = document.querySelector('input[name="food"]:checked').value;
     if (selectedButton === 'side') {
     dishPrinted.innerText = sides[getRandomIndex(sides)] + "!";
   } else if (selectedButton === 'main') {
     dishPrinted.innerText = mains[getRandomIndex(mains)] + "!";
   } else if (selectedButton === 'dessert') {
     dishPrinted.innerText = desserts[getRandomIndex(desserts)] + "!";
+  } else if (selectedButton === 'entire-meal') {
+    dishPrinted.innerHTML = `<p>You should make ${mains[getRandomIndex(mains)]} with a side of ${sides[getRandomIndex(sides)]} and ${desserts[getRandomIndex(desserts)]} for dessert!</p>`
   }
-  cookpotImage.classList.add('hidden');
```

Comment:
I broke this function up into two different functions to cut down on length. Want input from mentor about whether or not the showRandomDish function is still too long?